README.md

```diff
@@ -15,6 +15,36 @@ This project is under active development. The workflow code is being piloted in
 
 More documentation and samples will be coming soon!
 
+## Getting the library
+
+### Swift
+
+[![CocoaPods compatible](https://img.shields.io/cocoapods/v/Workflow.svg)](https://cocoapods.org/pods/Workflow)
+
+Just consume the workflow CocoaPods.
```

Comment:
@davidapgar PTAL

Reply:
Maybe something more like:
If you use CocoaPods to manage your dependencies, simply add Workflow and WorkflowUI to your Podfile:
```
pod 'Workflow'
pod 'WorkflowUI'
```

Reply:
Done.